devicetypes/stelpro/stelpro-ki-thermostat.src/stelpro-ki-thermostat.groovy

```diff
@@ -0,0 +1,572 @@
+/*
+ *  Copyright 2017 - 2018 Stelpro
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License. You may obtain a copy of the License at:
+ *
+ *	  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License
+ *  for the specific language governing permissions and limitations under the License.
+ *
+ *  Stelpro Ki Thermostat
+ *
+ *  Author: Stelpro
+ *
+ *  Date: 2018-04-24
+ */
+
+metadata {
+	definition (name: "Stelpro Ki Thermostat", namespace: "stelpro", author: "Stelpro", ocfDeviceType: "oic.d.thermostat") {
+		capability "Actuator"
+		capability "Temperature Measurement"
+		capability "Temperature Alarm"
+		capability "Thermostat"
```

Comment:
does having this cause problems since you have overlapping attributes?

Reply:
So far, it has worked.